src/ztc/datamodel/admin/forms.py

```diff
@@ -82,6 +84,17 @@ def _clean_selectielijstklasse(self):
             err = forms.ValidationError(msg, code='invalid')
             raise forms.ValidationError({'selectielijstklasse': err}) from exc
 
+        try:
+            # Check whether the url points to the correct resource
+            ResourceValidator(
+                'Resultaat',
+                'https://ref.tst.vng.cloud/referentielijsten/api/v1/schema/openapi.yaml?v=3',
```

Comment:
I'd like to see this defined as a constant at the top of the module rather than hardcoded here in the middle

Reply:
sorry - I wasn't clear enough. I meant the URL to the API spec, the rest of the validator can remain in the body of the method.